packages/components/src/internal/components/files/FileAttachmentForm.tsx

```diff
@@ -51,6 +51,7 @@ interface FileAttachmentFormProps {
     onCancel?: () => any;
     onFileChange?: (files: Map<string, File>) => any;
     onFileRemoval?: (attachmentName: string) => any;
+    importFieldsFromJson?: (file: File) => Promise<{success: boolean, msg?: string}>;
```

Comment:
should this return type use `SimpleResponse`?

Reply:
Yup! Done